lbrynet/core/LBRYMetadata.py

```diff
@@ -90,10 +90,10 @@ def _load_revision(self, version, f):
 
 
 class Metadata(dict):
-    def __init__(self, metadata):
+    def __init__(self, metadata, is_hex=False):
         dict.__init__(self)
         self.meta_version = None
-        metadata_to_load = deepcopy(metadata)
+        metadata_to_load = deepcopy(metadata if not is_hex else json.loads(metadata.decode("hex")))
```

Comment:
I'd move the logic for this into a class function.  Maybe:

```
@classmethod
def load_from_hex(cls, metadata):
    return cls(json.loads(metadata.decode("hex")))
```

In general `Boolean arguments loudly declare that the function does more than one thing. They are confusing and should be eliminated.`